add-mentee/index.ts

```diff
@@ -0,0 +1,10 @@
+import { AzureFunction } from "@azure/functions";
+import { container } from "tsyringe";
+import "reflect-metadata";
+import { setupContainer } from "../container";
+import { AddMentee } from "./add-mentee-handler";
+
+setupContainer();
+
```

Comment:
current work around - but will be replaced shortly since the new TS AzureFunctions helper has been released